test/unit/assorted/server_selection_logic_spec_utils.ts

```diff
@@ -0,0 +1,151 @@
+import { Document, EJSON } from 'bson';
+import { expect } from 'chai';
+import { readdirSync, readFileSync, statSync } from 'fs';
+import { basename, extname, join } from 'path';
+
+import {
+  ReadPreference,
+  ReadPreferenceMode,
+  ReadPreferenceOptions
+} from '../../../src/read_preference';
+import { ServerType, TopologyType } from '../../../src/sdam/common';
+import { ServerDescription, TagSet } from '../../../src/sdam/server_description';
+import * as ServerSelectors from '../../../src/sdam/server_selection';
+import { TopologyDescription } from '../../../src/sdam/topology_description';
+import { serverDescriptionFromDefinition } from './server_selection_spec_helper';
+
+interface ServerSelectionLogicTestServer {
+  address: string;
+  avg_rtt_ms: number;
+  type: ServerType;
+  tags?: TagSet;
+}
+interface Test {
```

Comment:
```suggestion
interface ServerSelectionTest {
```

Reply:
Renamed!  Let me know if you're okay with the name.